domain/status/service/service.go

```diff
@@ -132,78 +132,37 @@ type State interface {
 	DeleteUnitPresence(ctx context.Context, name coreunit.Name) error
 }
 
-// Service provides the API for working with the statuses of applications and units.
-type Service struct {
-	st            State
+// LeadershipService provides the API for working with the statuses of applications and units.
```

Comment:
This comment should be updated to reflect the new (sub-)service.

Reply:
Yep good spot